docs/dev/guide/contract-verification.md

```diff
@@ -35,11 +37,13 @@ Code verification requires six arguments:
 
 ## Modes of verification
 
-Depending on whether your contract is deployed on a network or with zkSync, there are two modes of contract verification:
+When your smart contract is deployed on zkSync, every zkSync user on the block explorer will be able to check the verified source code and interact with it.
+
```

Comment:
We have only one mode, so I would change the header to smth like "Source code privacy"